docs/guide-events.md

```diff
@@ -0,0 +1,55 @@
+---
+id: guide-events
+title: Considerations for fireEvent
+---
+
+## Interactions vs. events
+
+Based on [the Guiding Principles](guiding-principles.md), your test should
+resemble how users interact with your code (component, page, etc.) as much as
+possible. With this in mind, you should be aware that `fireEvent` is not actually
+how the user interacts with your application.
+
+Depending on the interaction multiple events can be fired in the browser and sometimes
+the events fired depend on the type of the element. For example consider a button
+element that should receive a keydown. It will only receive that event if the button
+is focused. If it is focused an additional click event is fired if the Enter key
+was pressed.
+
+You don't need to know every possible combination of interactions but you shouldn't
+have to much confidence in the interactive behavior of your components if you only
+test them with mocked sequences of events.
+
+## Alternatives
+
+We will describe a couple of simple adjustments to your tests that will increase your confidence
+in the interactive behavior of your components. For other interactions you should
+either consider using `user-event` or testing your components in a real environment
```

Comment:
Could we make `user-event` a link to the `user-event` docs page?

Reply:
done